compose/envoy/envoy.yaml

```diff
@@ -19,6 +19,16 @@ dynamic_resources:
 
 static_resources:
   clusters:
+
+    - name: httpbin
+      connect_timeout: 1s
+      type: logical_dns
+      lb_policy: round_robin
+      hosts:
+      - socket_address:
+          address: httpbin.org
```

Comment:
We should look into having this done inside `docker-compose` rather than using an external service. We have the `web` service inside compose which we can use, or we can create and use another one to distinguish it from the actual upstream service we want to authorize against.

Reply:
A new PR will be here in the following days, regarding JWT. Where cluster config has changed a bit, I'll make the change in that PR.